packages/util-utf8/src/toUtf8.browser.ts

```diff
@@ -0,0 +1,7 @@
+import { toUtf8 as jsToUtf8 } from "./pureJs";
+import { toUtf8 as textEncoderToUtf8 } from "./whatwgEncodingApi";
+
+declare const TextDecoder: Function | undefined;
+
+export const toUtf8 = (input: Uint8Array): string =>
+  typeof TextDecoder === "function" ? textEncoderToUtf8(input) : jsToUtf8(input);
```

Comment:
TextDecoder is supported in all browsers which support ES2020
We can remove `jsToUtf8`, and copy `textEncoderToUtf8` code inside this function

MDN: https://developer.mozilla.org/en-US/docs/Web/API/TextDecoder#browser_compatibility
PR which bumped ES version to ES2020: https://github.com/aws/aws-sdk-js-v3/pull/4005

Reply:
fixed